### ISSUE TITLE
<< and >> not implemented properly

### ISSUE THREAD

**Issue by Sp3000** [cheddar-lang/Cheddar#50]
**Version:** 0.3.0

### Problem

```
cheddar> 1<<2
 `<<` has no behavior for types `nil` and `Number`
cheddar> 1>>2
 `>>` has no behavior for types `nil` and `Number`
```

### Desired Behavior

Bit shift? Or whatever it's meant to implement.

**Comment by vihanb** [comment]
yeah bit shits 
(hue hue hue i made bit shit joke for the 1000th time)

**Comment by vihanb** [comment]
Fixed in 039b60c
